src/data/value.ts

```diff
@@ -444,6 +554,8 @@ export namespace TransferableValues {
                 return result;
             case "link":
                 return { "___transfer-type": "link", value: wrapped.value.toObject() };
+            case "task":
+                return { "___transfer-type": "task", value: transferable(wrapped.value.toObject()) };
```

Comment:
This required wrapping because `toObject` returns `LiteralValue`s rather than `Transferable`

Reply:
Ohhhhhh, yes, this makes sense. This is correct.